cmd/node/config/systemSmartContractsConfig.toml

```diff
@@ -8,8 +8,8 @@
     NumRoundsWithoutBleed = 100
     MaximumPercentageToBleed = 0.5
     BleedPercentagePerRound = 0.00001
-    MaxNumberOfNodesForStake = 28
-    NodesToSelectInAuction = 28
```

Comment:
Moved if back to 28

Reply:
@LucianMincu needs to test this in "as close as it can get to mainnet" scenario